docs/modules/ROOT/pages/explanations/decisions/subscription-tracking.adoc

```diff
@@ -0,0 +1,74 @@
+= Subscription tracking
+
+== Problem
+
+We need to track the yearly subscriptions bought for OpenShift clusters.
+The exact SKU count is required for a monthly report to Red Hat.
+Red Hat OpenShift SKU subscriptions are based on cores or vCPU count (called `CORE BAND`), in our case we use vCPU count.
+We need to track how many long-term SKU subscriptions (Yearly or 3 Years) we bought, in which timeframe they're valid (there can be multiple ranges, depending on when they where bought) and how many are assigned.
+And with that information, we also need to report on the difference so buy additional monthly SKUs, should there be a difference.
+
+A monthly usage report has to be sent to Red Hat, detailing the use of SKUs, assigned to consumers of them (our customers).
+
+=== Goals
+
+* Long-term SKU subscriptions (Yearly or 3 Years) are tracked in a central place, including validity information
+* Detailed SKU subscription count is available for an automated monthly report to Red Hat
+* A suggestion for how many yearly subscriptions to buy is available to the account or partner manager
+
+=== Non-goals
+
+* Full automation of subscription buying
+
+== Proposals
+
+=== Option 1: Database and Web UI (custom/ Grafana)
+
+We store subscriptions in a database and build a custom UI or Grafana dashboard to visualize the data.
+
+This allows instant feedback and would allow account managers to track subscriptions themselves without having to ask our team.
+
+Building the UI would take time and we would need to maintain it.
+
+Grafana most likely doesn't support editing data.
+
+=== Option 2: Store in ConfigMap managed by Commodore
+
+We store the bought subscriptions in a ConfigMap managed by Commodore.
+This gives us free versioning and a history of changes.
+Coupled with a Jira ticket we can track who bought and licenses for which customer.
```

Comment:
This sentence doesn't make click in my head. :thinking:

Reply:
fixed the typo